### DISCUSSION TITLE
Suggestion for a copy option

### DISCUSSION THREAD

**Discussion by FabriceDB** [rappen/FetchXMLBuilder#629] [Ideas]
When selecting a cell in a resultset, it's only possible to do a Ctrl-C, and we get in the clipboard the name of the column and the value of the cell. So, it's impossible to do a direct copy-paste  from the cell to a filter in the app. What should be fine is to add a popup menu in the grid, with 2 options **copy** and **copy with column name**, as in sql server manager studio.
![image](https://user-images.githubusercontent.com/90260514/151124255-9d887b4a-5511-4ea8-8e29-04d53fc58456.png)

Great tool!

**Comment by rappen** [comment]
You can!!
See #625 😉

**Reply by FabriceDB** [reply]
omg, sorry about that, I didn't see this options zone. It's not usual, this design. Adding the options in a popup, or in the main option menu could maybe be more friendlier for new users (now I know it's there, I don't have to search it again)

**Reply by rappen** [reply]
Good idea - new issue added! #630 🙂
